Add debug output because of test failures causing FTBFS on kfreebsd-i386
Index: apr/test/testlock.c
===================================================================
--- apr.orig/test/testlock.c	2012-05-28 13:08:20.394496385 +0200
+++ apr/test/testlock.c	2012-05-28 13:09:00.902494363 +0200
@@ -297,6 +297,7 @@
             continue;
         }
         ABTS_INT_EQUAL(tc, 1, APR_STATUS_IS_TIMEUP(s));
+        printf("Timer returned %dms late\n", (int)(end - begin - timeout)/1000);
         ABTS_ASSERT(tc, "Timer returned too late", end - begin - timeout < 100000);
         break;
     }
